fastlane/Fastfile

```diff
@@ -95,12 +95,20 @@ end
 desc 'Update changelog in iTunes Connect with the content from Docs/NEWS'
 private_lane :update_changelog do |options|
   platform = options[:platform]
-  # Splits the News by -------- get out the top notes
-  changelog = File.read('../Docs/NEWS').split('-----------')[1].split('-----------').first
-  temp_changelog = changelog.split(platform)
-  temp_changelog = changelog.split('tvOS') if temp_changelog.count <= 1
-  changelog = temp_changelog[0..-2].join.strip
-  set_changelog(app_identifier: 'org.videolan.vlc-ios', changelog: changelog, username: '*', team_name: 'VideoLAN')
+  version = options[:version]
+
+  regex = /## #{platform} \[#{version}\](.*?)^##/m
```

Comment:
that is sooo much better than what I had!!

Reply:
I cheated changing the format of the changelog, the previous format was much harder the parse 😉